### DISCUSSION TITLE
Importing Device Configuration Error

### DISCUSSION THREAD

**Discussion by shabsn** [Micke-K/IntuneManagement#247] [General]
Hey, I am trying to import Device Configurations from the Open Intune Baselines but I am getting this error:

![image](https://github.com/Micke-K/IntuneManagement/assets/173994324/c7026605-4dd8-40fe-98ee-90a963ecc8de)

Can you maybe help with that?

Thank you :)

**Comment by AntonWilloughby** [comment]
Ignore my last response as OAuth permission errors are in yellow :)

I was able to replicate the issue and it seems you need to ensure the "Import folder" is pointing to the location of your .json files or else it seems to try and use the "Migration table":

Notice in the first screenshot, I do not have any visible .json files? 

![image](https://github.com/Micke-K/IntuneManagement/assets/116081795/b0975d30-7c75-4cc7-9ab6-a96256939bc1)

But by changing the import folder to the location of my .json files, the app then loads them into the window and you can then select which one you would like to import:


![image](https://github.com/Micke-K/IntuneManagement/assets/116081795/8698fdef-141f-4fd2-b16f-6278ee74b7d3)

Select the checkbox(es) and then hit the import button and hopefully your config appears in your Intune portal :)
